src/test/java/core/annotation/AnnotationProcessorTest.java

```diff
@@ -136,7 +135,8 @@ public void testFindAnnotationMultipleMatches()
      */
     @Test(expected = AnnotationProcessor.NoAnnotationsFoundException.class)
     public void testFindAnnotationNoMatch()
-            throws AnnotationProcessor.TooManyAnnotationsFoundException, AnnotationProcessor.NoAnnotationsFoundException {
+            throws AnnotationProcessor.TooManyAnnotationsFoundException,
```

Comment:
I guess since this error never occurs, and a user do not want to see any errors in the console, we should remove this